moskito-frontend/src/components/Button/Button.js

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+Button.propTypes = {
+    children: PropTypes.string.isRequired
+}
+
+export default function Button({children}) {
+    return (
+        <MoskitoButton>{children}</MoskitoButton>
+    )
+}
+
+const MoskitoButton = styled.button`
+    padding: 1em 2em;
+    border-radius: 10px;
+    width: auto;
+    border: none;
+    background: #c2d6d3;
+    box-shadow: 2px 4px 8px -4px rgba(0,0,0,0.6), 
+                -2px -2px 2px 0px rgba(255,255,255,0.5);
+    color: #476c85;
+    text-transform: uppercase;
+    margin: 3em auto;
+
+    &:hover,
+    &:focus{
+        background: linear-gradient(145deg, #A2B2B0, #E2FAF6);
+        box-shadow: 2px 4px 8px -4px rgba(0,0,0,0.6), 
+                -2px -2px 2px 0px rgba(255,255,255,0.5);
+        outline: none;
+
+    }
```

Comment:
Just a thought on naming the component 🙂: you could exchange Button and MoskitoButton, since right now Button (which is more general) returns the MoskitoButton (which is more specific). This way you could add more buttons to Button.js for export. If there will be only one button, I would name the file MoskitoButton.js and keep the default export.

```suggestion
export function MoskitoButton({children}) {
    return (
        <StyledButton>{children}</StyledButton>
    )
}

const StyledButton = styled.button`
    padding: 1em 2em;
    border-radius: 10px;
    width: auto;
    border: none;
    background: #c2d6d3;
    box-shadow: 2px 4px 8px -4px rgba(0,0,0,0.6), 
                -2px -2px 2px 0px rgba(255,255,255,0.5);
    color: #476c85;
    text-transform: uppercase;
    margin: 3em auto;

    &:hover,
    &:focus{
        background: linear-gradient(145deg, #A2B2B0, #E2FAF6);
        box-shadow: 2px 4px 8px -4px rgba(0,0,0,0.6), 
                -2px -2px 2px 0px rgba(255,255,255,0.5);
        outline: none;

    }
```